### ISSUE TITLE
Possibility to add Introduction view inside of list

### ISSUE THREAD

**Issue by hiddevdploeg** [AvdLee/Roadmap#5]
Being able to add an introduction view to the roadmap list can give the user some context on what they’re looking at

**Comment by jordibruin** [comment]
<img width="673" alt="image" src="https://user-images.githubusercontent.com/170948/220324396-3668a36f-3b42-4596-b03d-203b5622b3ca.png">

Something similar to this. Not sure how antoine implemented this now.

**Comment by nickkohrn** [comment]
I did something similar by using the following logic:

```swift
NavigationLink {
    VStack(spacing: 0) {
        Text("Below are some features coming to {app}. Please vote on the things that are most important to you and help shape the roadmap for the {app}'s future.")
            .font(.footnote)
            .padding()
        RoadmapView(configuration: viewStore.roadmapConfiguration)
    }
    .navigationBarTitle("Roadmap")
    .navigationBarTitleDisplayMode(.inline)
} label: {
    Label {
        Text("Roadmap")
    } icon: {
        Image(systemName: "map")
    }
}
```

<details>
    <summary>See Screenshot</summary>

![IMG_1570 2](https://user-images.githubusercontent.com/5864877/220439810-85961199-e807-43b1-ada8-5dc222ac7116.jpg)
    
</details>

The only downside is that the `Text` above the `RoadmapView` does not scroll. This causes large type sizes to severely restrict the amount of available space for the feature list to be displayed within.

<details>
    <summary>See Recording</summary>
   
https://user-images.githubusercontent.com/5864877/220441216-9785cb08-b48a-42c4-849e-cbaaa827354c.MP4

</details>

**Comment by AvdLee** [comment]
> The only downside is that the Text above the RoadmapView does not scroll. This causes large type sizes to severely restrict the amount of available space for the feature list to be displayed within.

Yes, my implementation comes with the same limitation. My code looks as follows:

```swift
@available(macOS 13.0, *)
struct RoadmapContainerView: View {
    let configuration: RoadmapConfiguration

    var body: some View {
        VStack(spacing: 0) {
            Text("RocketSim Roadmap")
                .font(.headline)
            GroupBox {
                Text("These are features that are coming to RocketSim soon. You can vote on the ones you'd love to see me add first. [Let me know](https://github.com/AvdLee/RocketSimApp/issues/new?assignees=AvdLee&labels=enhancement&template=feature_request.md) if you have a suggestion for a feature that needs to be added to this list.")
                    .padding(10)
            }.padding(.horizontal, 20)
                .padding(.vertical, 20)
            Divider()
                .padding(.horizontal, 20)
            RoadmapView(configuration: configuration)
                .introspectTableView(customize: { tableView in
                    tableView.enclosingScrollView?.automaticallyAdjustsContentInsets = false
                    tableView.enclosingScrollView?.contentInsets = .init(top: 20, left: 10, bottom: 20, right: 10)
                })
                .scrollContentBackground(.hidden)
        }
    }
}
```

I can totally see us supporting something like `header: () -> some View` that would scroll with the features, similar to the SwiftUI Form Section header API.
